src/rprblender/engine/engine.py

```diff
@@ -83,28 +83,25 @@ def cache_blur_data(self, depsgraph: bpy.types.Depsgraph):
         position = scene.cycles.motion_blur_position
 
         if position == 'END':  # shutter closes at the current frame, so [N-1 .. N]
-            start_frame = scene.frame_current - 1
-            subframe = 0.0
+            from_frame = (scene.frame_current, 0.0)
```

Comment:
wouldn't this make motion backwards?

Reply:
Yes it would, but seems this is exactly what user expects for this option